### PR TITLE
Clean up for the new domain name changes.

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:* [15877](https://github.com/bcgov/entity/issues/15877)

*Description of changes:*
1. Clean up unused files;
2. Enable cloud build PR review feature;
3. Move firebase and cloud build configuration to UI CD flow in bcregisty-sre repo;
4. Update environment variables' value;

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the bcrs-entities-create-ui license (Apache 2.0).

### CODE REVIEW

**Comment by pwei1018** [comment]
Temporary Url for review: https://business-edit-dev--pr-486-3tpye30o.web.app

File: devops/cloudbuild-pr.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      #!/usr/bin/env bash
+      cp .env.example .env
+
+  #
+  # Generate the static site
+  #
+  - name: node:16.14.2
+    entrypoint: npm
+    env:
+      - 'VVUE_APP_BUSINESS_EDIT_LD_CLIENT_ID=$_VUE_APP_BUSINESS_EDIT_LD_CLIENT_ID'
```
Comment (severinbeauvais) [inline devops/cloudbuild-pr.yaml:19]: Should be BUSINESS_CREATE.

**Review by severinbeauvais — APPROVED** [review]
Approved but please fix keys before merging.

**Review by ketaki-deodhar — APPROVED** [review]

File: devops/cloudbuild-pr.yaml (hunk 1 above)
Comment (severinbeauvais) [reply devops/cloudbuild-pr.yaml:19]: My mistake. Resolving this conversation.

**Pull request merged**